dask/templates/dask-jupyter-deployment.yaml

```diff
@@ -28,6 +28,15 @@ spec:
     spec:
       imagePullSecrets:
         {{- toYaml .Values.jupyter.image.pullSecrets | nindent 8 }}
+      {{- if .Values.jupyter.persistent.enable }}
+      initContainers:
+        - name: {{ .Release.Name }}-jupyter-volume-permissions
+          image: busybox:1.31.1
+          command: ["sh", "-c", "chmod -Rv 777 /data/ && chown 0:100 -R /data/"]
```

Comment:
Setting permissions to 777 is generally a bad idea. You may find 755 is more appropriate.

However could you provide a little explaination why this is necessary at all?

Reply:
The storage class that I am using is longhorn.  The volumes they give you are owned by root by default. That init container sets the permissions so that the jovyan user can read and write to the PV. 

I'm not aware of any storage classes that let you explicitly set the default ownership. I will add an option so its easy to disable this init container if desired.